mcstatus/querier.py

```diff
@@ -58,6 +55,45 @@ def handshake(self) -> None | Awaitable[None]:
     def read_query(self) -> QueryResponse | Awaitable[QueryResponse]:
         raise NotImplementedError
 
+    def _parse_response(self, response: Connection) -> tuple[RawQueryResponse, list[str]]:
+        """Transform the connection object (the result) into dict which is passed to the QueryResponse constructor.
+
+        :return: A tuple with two elements. First is `raw` answer and second is list of players.
+        """
+        response.read(len("splitnum") + 1 + 1 + 1)
```

Comment:
I see that this was copied from the original code, but I'm curious why it used `+ 1 + 1 + 1` instead of just `+ 3`.

Same question for a similar statement below:
```python
response.read(len("player_") + 1 + 1)
```

If we can't figure out a reason, we might as well change it, since it's quite weird.

Reply:
If only I knew... Changed it to 3